src/main/java/com/jcabi/github/mock/MkIssueLabels.java

```diff
@@ -154,20 +178,31 @@ public Label map(final XML xml) {
     public void remove(@NotNull(message = "name cannot be NULL")
         final String name
     ) throws IOException {
-        this.storage.apply(
-            new Directives().xpath(
-                String.format("%s/label[.='%s']", this.xpath(), name)
-            ).remove()
-        );
+        if (this.labelSet().contains(name)) {
+            this.storage.apply(
+                new Directives().xpath(
+                    String.format("%s/label[.='%s']", this.xpath(), name)
+                ).remove()
+            );
+            final MkIssueEvents events = new MkIssueEvents(
```

Comment:
@cvrebert No need to assign it to a variable, just inline the `create` call below.